app/controllers/api/v1/follows_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::V1::FollowsController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
This is *basically* OK, but I will confess for a personal preference for `protect_from_forgery unless: -> { request.format.json? }` instead. But that implementation requires more work in your `fetch` requests, so this works too.